Classes/Security/CheckPermissions.php

```diff
@@ -198,7 +198,17 @@ public function getFolderPermissions(FolderInterface $folder)
     }
 
     /**
-     * Get FeGroups that are allowed to view a file/folder (checks full rootline)
+     * Get FeGroups that are allowed to view a file/folder (checks NOT full rootline)
+     * Check from the given folder up to root, i. e. the reverse! rootline. 
+     * First restriction matches.
+     * 
+     * This Bugfix should be sure because it's only called from:
```

Comment:
Please remove the comments about the bug fix here.

Reply:
Ok, I'll do it